### PR TITLE
Windows assetpack download script

### PR DESCRIPTION
**Pull request by azaleacolburn**
### Description
Created assetpack download script for windows users using run-script-os

### Objectives
- [x] Assetpack download script attempt 1

### Testing Done
- [x] Test download script on windows computer (Brandon | Ayla)

[JIRA Issue](https://jira.autodesk.com/browse/AARD-1715)

### CODE REVIEW

File: fission/package.json [hunk 1]
```diff
@@ -1,73 +1,73 @@
...
+        "preview": "vite preview --base=/fission/",
+        "test": "vitest",
+        "lint:fix": "npm run lint --fix",
+        "prettier": "npx prettier src --check",
+        "prettier:fix": "npm run prettier --write",
+        "format": "npm run prettier:fix && npm run lint:fix",
+        "build:prod": "tsc && vite build --base=/fission/",
+        "assetpack": "run-script-os",
+        "assetpack:nix": "curl -o public/assetpack.zip https://synthesis.autodesk.com/Downloadables/assetpack.zip && unzip -o public/assetpack.zip -d public/",
+        "assetpack:win32": "curl -o public/assetpack.zip https://synthesis.autodesk.com/Downloadables/assetpack.zip && tar -xf public/assetpack.zip"
```
Comment (BrandonPacewic) [inline fission/package.json]:
```suggestion
        "assetpack:win32": "curl -o public/assetpack.zip https://synthesis.autodesk.com/Downloadables/assetpack.zip && tar -xf public/assetpack.zip -C public/"
```

Should be placed into the `public/` dir after unzip.

**Review by BrandonPacewic — CHANGES_REQUESTED** [review]
Also we should maintain the 2 space formatting for all `.json` files.

**Review by BrandonPacewic — APPROVED** [review]
Looks like there were still some files that were not up to date with the new formatting.

Download and unzip works on my Windows machine.

LGTM!

**Review by HunterBarclay — APPROVED** [review]
Works great on my windows machine
👍

**Pull request merged**